src/main/java/io/cdap/plugin/Profile.java

```diff
@@ -56,4 +56,12 @@ protected double V(double value) {
   }
 
   public abstract void results(StructuredRecord.Builder builder);
+
+  public long getNulls() {
+    return 0;
+  }
+
+  public long getCount() {
+    return 0;
+  }
```

Comment:
`nulls` and `count` is very specific to `Categorical` profile, these methods should not be included here.

For the unit test perspective, we can verify the expected `StructuredRecord` instead of introducing new public methods.